internal/controller/daemonset/instaslice_daemonset_test.go

```diff
@@ -198,7 +198,7 @@ func newInstaslice(name, podUUID string, status inferencev1alpha1.AllocationStat
 	return instaslice
 }
 
-func Test_calculateTotalMemoryGB(t *testing.T) {
+func Test_calculateEmulatedTotalMemoryGB(t *testing.T) {
```

Comment:
Maybe it would be a good idea to stay with golang convention and name it `TestCalculateEmulatedTotalMemoryGB` ?